cluster/config-defaults.yaml

```diff
@@ -511,6 +525,18 @@ audittrail_url: ""
 {{end}}
 audittrail_root_account_role: ""
 
+# kube-apiserver / kube-proxy container image name.
+# Temporarily added to help during the Kubernetes service network range migration.
+kube_apiserver_container_image_name: registry.opensource.zalan.do/teapot/kube-apiserver-internal:v1.21.12-pr-71-5
+kube_proxy_container_image: registry.opensource.zalan.do/teapot/kube-proxy-internal:v1.21.12-pr-71-2
```

Comment:
These must be the images built from master, maybe align with @linki to ensure it's the right multi-arch versions we use.

Reply:
e2e tests fixed after git rebase dev